.github/workflows/ci.yml

```diff
@@ -27,7 +30,7 @@ jobs:
 
     - name: Lint with flake8
       run: |
-        flake8 . --count --show-source --statistics
+        flake8 fpga_topwrap --count --show-source --statistics
```

Comment:
I think this should be a separate commit (and the change of the trigger above too)

Reply:
This change is a separate commit now. The whole CI for GH Pages is a new file in a separate commit, too.